content/ja/docs/concepts/containers/images.md

```diff
@@ -2,245 +2,362 @@
-- `imagePullPolicy`フィールドを省略し、コンテナイメージのタグを指定しなかった場合、`imagePullPolicy`には自動的に`Always`が設定される
-- `imagePullPolicy`フィールドを省略し、コンテナイメージのタグに`:latest`以外を指定した場合、`imagePullPolicy`には自動的に`IfNotPresent`が設定される
+- `imagePullPolicy`フィールドを省略し、かつコンテナイメージにダイジェストを指定した場合、`imagePullPolicy`には自動的に`IfNotPresent`に設定される。
+- `imagePullPolicy`フィールドを省略し、コンテナイメージのタグに`:latest`を指定した場合、`imagePullPolicy`には自動的に`Always`が設定される。
+- `imagePullPolicy`フィールドを省略し、コンテナイメージのタグを指定しなかった場合、`imagePullPolicy`には自動的に`Always`が設定される。
+- `imagePullPolicy`フィールドを省略し、コンテナイメージのタグに`:latest`以外を指定した場合、`imagePullPolicy`には自動的に`IfNotPresent`が設定される。
 
 {{< note >}}
-コンテナの`imagePullPolicy`の値は、そのオブジェクトが最初に _作成_ されたときに常に設定され、イメージのタグが後で変更された場合でも更新されません。
+コンテナの`imagePullPolicy`の値は、そのオブジェクトが最初に _作成_ されたときに常に設定され、その後イメージのタグやダイジェストが変更されても更新されません。
 
-例えば、タグが`:latest` _でない_ イメージを使ってDeploymentを生成した場合、後でDeploymentのイメージを`:latest`タグに変更しても、`imagePullPolicy`は`Always`に更新されません。オブジェクトのPullポリシーは、初期作成後に手動で変更する必要があります。
+たとえば、タグが`:latest` _でない_ イメージを使ってDeploymentを作成した場合、後でDeploymentのイメージを`:latest`タグに変更しても、`imagePullPolicy`は`Always`に更新 _されません_。初回作成後にプルポリシーを変更したい場合は、手動で更新しなければいけません。
 {{< /note >}}
 
-#### 必要なイメージをPullする
+#### イメージ取得を強制する
 
-常に強制的にPullしたい場合は、以下のいずれかを行ってください。
+常に強制的に取得したい場合は、以下のいずれかを実行できます:
 
 - コンテナの`imagePullPolicy`に`Always`を設定する。
-- `imagePullPolicy`を省略し、使用するイメージに`:latest`タグ使用する。Pod生成時に、Kubernetesがポリシーに`Always`を設定する。
-- `imagePullPolicy`と使用するイメージのタグを省略する。Pod生成時に、Kubernetesがポリシーに`Always`を設定する。
+- `imagePullPolicy`を省略し、使用するイメージのタグに`:latest`を指定する(KubernetesはPodを送信する際にポリシーを`Always`に設定します)。
+- `imagePullPolicy`と使用するイメージのタグを省略する(KubernetesはPodを送信する際にポリシーを`Always`に設定します)。
 - [AlwaysPullImages](/docs/reference/access-authn-authz/admission-controllers/#alwayspullimages)アドミッションコントローラーを有効にする。
 
 ### ImagePullBackOff
 
-kubeletがコンテナランタイムを使ってPodのコンテナの生成を開始するとき、`ImagePullBackOff`のためにコンテナが[Waiting](/ja/docs/concepts/workloads/pods/pod-lifecycle/#container-state-waiting)状態になる可能性があります。
+kubeletがコンテナランタイムを使ってPodのコンテナの作成を開始するとき、`ImagePullBackOff`のためにコンテナが[Waiting](/ja/docs/concepts/workloads/pods/pod-lifecycle/#container-state-waiting)状態になる可能性があります。
 
-`ImagePullBackOff`ステータスは、KubernetesがコンテナイメージをPullできないために、コンテナを開始できないことを意味します(イメージ名が無効である、`imagePullSecret`なしでプライベートレジストリからPullしたなどの理由のため)。`BackOff`は、バックオフの遅延を増加させながらKubernetesがイメージをPullしようとし続けることを示します。
+`ImagePullBackOff`状態は、Kubernetesがコンテナイメージを取得できず、コンテナを開始できないことを意味します(イメージ名が無効である、`imagePullSecret`無しでプライベートレジストリから取得したなどの理由のため)。`BackOff`は、バックオフの遅延を増加させながらKubernetesがイメージを取得しようとし続けることを示します。
 
 Kubernetesは、組み込まれた制限である300秒(5分)に達するまで、試行するごとに遅延を増加させます。
 
-## イメージインデックスを使ったマルチアーキテクチャイメージ
+### ランタイムクラスごとのイメージ取得
+
+{{< feature-state feature_gate_name="RuntimeClassInImageCriApi" >}}
+Kubernetesには、PodのRuntimeClassに基づいてイメージを取得するアルファ機能のサポートが含まれています。
+
+`RuntimeClassInImageCriApi`[フィーチャーゲート](/ja/docs/reference/command-line-tools-reference/feature-gates/)を有効にすると、kubeletはイメージ名やダイジェストだけでなく、イメージ名とランタイムハンドラーのタプルでコンテナイメージを参照するようになります。
```

Comment:
```suggestion
`RuntimeClassInImageCriApi`[フィーチャーゲート](/ja/docs/reference/command-line-tools-reference/feature-gates/)を有効にすると、kubeletは単にイメージ名やダイジェストではなく、イメージ名とランタイムハンドラーのタプルでコンテナイメージを参照するようになります。
```
A rather than Bは、「BではなくA」や「BよりむしろA」という訳し方が適切だと思います。

> ... a tuple of image name and runtime handler rather than just the image name or digest.